core/src/date-input/date-input.tsx

```diff
@@ -113,6 +113,20 @@ const getDisabledDays = (props: Props): Modifier => {
 	return days;
 };
 
+/**
+ * Date Inputs let users enter a date, by either typing or choosing from a
+ * pop-up calendar. They are an alternative to `<Input type="date"\>` that
```

Comment:
You mean "There are.." right?

Reply:
No, I do mean "They are". "They" here mean the "Date Inputs", which are alternatives to the built in one.

But anyway, I think "They are an alternative" sounds wrong. Should be "They are alternatives to ..."

Reply:
@ng-hai can you check again? I've just updated it to be more clear

Reply:
Done